api/src/main/java/org/allaymc/api/item/component/ItemBaseComponent.java

```diff
@@ -619,4 +620,8 @@ default boolean hasBlockEntityNBT() {
      * @param lockMode The lock mode of the item.
      */
     void setLockMode(ItemLockMode lockMode);
+
+    default boolean isAir() {
```

Comment:
I think this method could lead misunderstanding easily, maybe choose another method name?